client/components/ovh-task-follow/modal/ovh-task-follow-modal.controller.js

```diff
@@ -0,0 +1,29 @@
+angular.module("managerApp").controller("ovhTaskFollowModalCtrl", function ($scope, $uibModalInstance) {
+    "use strict";
+
+    var self = this;
+
+    self.isOpenState = {};
+    self.tasks = null;
+
+    /*= =============================
+    =            EVENTS            =
+    ==============================*/
+
+    self.cancel = function () {
+        $uibModalInstance.dismiss();
+    };
+
+    /* -----  End of EVENTS  ------*/
+
+    /*= =====================================
+    =            INITIALIZATION            =
+    ======================================*/
+
+    self.$onInit = function () {
+        self.data = $scope.$parent.$ctrl.data;
```

Comment:
jleveugle 9 days ago  Member
What do you think about avoiding using $scope ?

I hope one day, we will just do a breaking release and pass all our component to ng4+ :)

@jleveugle
jleveugle 9 days ago  Member
We should have a look to : https://github.com/johnpapa/angular-styleguide/blob/master/a1/README.md#controlleras-controller-syntax